### ISSUE TITLE
Only the last character of query parameters end up in Node handler

### ISSUE THREAD

**Issue by nickgravelyn** [awslabs/aws-sam-cli#827]
**Description:**

When trying to access query parameters in my Node handler, only the last character is given to me in the `event.queryStringParameters`.

**Steps to reproduce the issue:**
1. Create a new SAM app with this `template.yaml`:

```yaml
AWSTemplateFormatVersion: '2010-09-09'
Transform: AWS::Serverless-2016-10-31

Resources:
  HelloWorldFunction:
    Type: AWS::Serverless::Function
    Properties:
      CodeUri: .
      Handler: index.lambdaHandler
      Runtime: nodejs8.10
      Events:
        HelloWorld:
          Type: Api
          Properties:
            Path: /
            Method: get
```

and this `index.js` file

```javascript
exports.lambdaHandler = async event => ({
  statusCode: 200,
  body: JSON.stringify(event.queryStringParameters)
});
```

2. Start the API with `sam local start-api`.
3. Access the endpoint with some query parameters, e.g. `curl http://localhost:3000/?hello=world`

**Observed result:**

The endpoint returns `{"hello":"d"}`

**Expected result:**

The endpoint returns `{"hello":"world"}`

**Additional environment details (Ex: Windows, Mac, Amazon Linux etc)**

macOS Mojave

**Output of `sam --version`**:

SAM CLI, version 0.6.1

**Optional Debug logs**:

Nothing much useful in there other than to note that it does log the full correct URL with query parameters:

```
2018-12-04 09:05:49 127.0.0.1 - - [04/Dec/2018 09:05:49] "GET /?hello=world HTTP/1.1" 200 -
```

**Comment by jfuss** [comment]
Please make sure you search closed issue.

Duplicate of #736 and released in 0.6.2

**Comment by nickgravelyn** [comment]
Oh whoops. I searched and didn't realize it only searched open issues. Thanks!
